src/integTestFixtures/groovy/org/gradle/playframework/fixtures/app/WithFailingTestsApp.groovy

```diff
@@ -12,7 +12,7 @@ class WithFailingTestsApp extends PlayApp {
 
     @Override
     SourceFile getGradleBuild() {
-        def gradleBuild = sourceFile("", "build.gradle", "basicplayapp")
+        def gradleBuild = sourceFile("", "build.gradle.ftl", "basicplayapp")
```

Comment:
Can you explain why this is named with the extension `.ftl`?

Reply:
NVM, I understand, you're using freemarker.

Reply:
Yep, `.ftl` is one of the recommended file extensions for a FreeMarker template.  It doesn't really buy much with the IDE, but the test fixtures code checks for the extension to know if it should run the file through the FreeMarker engine.